### ISSUE TITLE
Re-purpose one of analog inputs for 2nd channel hall input

### ISSUE THREAD

**Issue by rusefi** [rusefi/hw_microRusEfi#111]
at the moment we have one Hall channel and one channel which could be either VR or hall depending on what resistors are mounter.

This hardware flexibility is quite labor intense, a nicer experience could be having one more hall-only channel and leave VR channel as VR only.

This would require selecting one more ICU input pin. We totally have ICU pin options (do we?)

And technically we can confirm PAL input mode.

**Comment by rusefi** [comment]
TL;DR -  the second hard-wired hall channel should be PA7 / current MRE AV3


VR input is routed to PC6 which is stm32 TIM3 or TIM8
Existing Cam is routed to PA5 which is TIM2
Following TIMs are available for ICU:

#define STM32_ICU_USE_TIM1                  TRUE
#define STM32_ICU_USE_TIM2                  TRUE
#define STM32_ICU_USE_TIM3                  TRUE
#define STM32_ICU_USE_TIM8                  TRUE
#define STM32_ICU_USE_TIM9                  TRUE

between ICU and PAL modes conflict avoidance PA7 is the best candidate to be second Hall hard-wired channel: this case PC6 could be shifted to use TIM8, PA7 would use TIM3 - so we have three ICU channels not conflicting.

Also PAL would allow PA5 PC6 and PA7 to operated simultaneously if needed.
